akka-bench-jmh/src/main/scala/akka/actor/ForkJoinActorBenchmark.scala

```diff
@@ -180,14 +195,18 @@ class ForkJoinActorBenchmark {
 object ForkJoinActorBenchmark {
   case object Stop
   case object Message
-  final val timeout = 15.seconds
-  final val messages = 400000
+  final val timeout = 30.seconds
+  final val messages = 2000000 // messages per actor pair
 
+  // Constants because they are used in annotations
   // update according to cpu
-  final val cores = 8
-  // 2 actors per
-  final val moreThanCoresActorPairs = cores * 2
-  final val lessThanCoresActorPairs = (cores / 2) - 1
+  final val cores = 24
+  final val coresStr = "24"
+  final val cores2xStr = "48"
+  final val cores4xStr = "96"
+  // 2 actors per pair
+  final val moreThanCoresActorPairs = cores
+  final val lessThanCoresActorPairs = cores / 4
   final val totalMessagesMoreThanCores = moreThanCoresActorPairs * messages
   final val totalMessagesLessThanCores = lessThanCoresActorPairs * messages
   final val totalMessagesSameAsCores = cores * messages
```

Comment:
Very good observation. 

Each `PingPong` actor stops after `messages/2`, i.e. a pair of `PingPong` actors process `messages` together.

For the `SameAsCores` case we have `cores` actors, i.e. `cores/2` pairs.